### ISSUE TITLE
[Keyboard navigation-Help]Role property is not defined for Help control.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2539]
**User Experience:**
If any control does not have role  then the person with visual related disability will find it difficult to interpret the proper fields information. This will hinder him to access all the controls of the page efficiently and also not compatible with AT.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Prerequisites:** Turn on Narrator

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Microbit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section
4-Verify, If role property is defined for help control or not.

**Actual Result**
Role property is not defined  for Help control. Due to which narrator reads only name of control.

**Expected Result**
Role property i.e. "Menu button" should be defined  for Help control. 

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Suggested fix**
Using ARIA role to expose the control type of user interface component
e.g.
< div role="toolbar" tabindex="0" id="customToolbar" >
< img src="img/btn1.gif" role="button" tabindex="-1" alt="Home" id="b1" title="Home" >

Please Refer attachment for more details
![mas4 1 2_role is not defined for help](https://user-images.githubusercontent.com/30459830/28661752-518d9168-72d5-11e7-942e-593fadd5aaa6.png)

**Comment by veler** [comment]
This issue has already been fixed by the past. Please try to reproduce it again and be sure to be on [https://makecode.microbit.org/acc](https://makecode.microbit.org/acc), not [https://makecode.microbit.org/](https://makecode.microbit.org/).

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug.
